src/FlaUI.Core/Application.cs

```diff
@@ -13,7 +13,7 @@ namespace FlaUI.Core
 {
     public class Application : IDisposable
     {
-        private static readonly ILogger Log = new ConsoleLogger();
+        public ILogger Log { get; }

```

Comment:
Is this even needed? We anyway use the Logger.Default. I think allowing different loggers for different classes would be an overkill.